scripts/setup_plugin_repos.py

```
@@ -0,0 +1,146 @@
+        if link_path.exists() or link_path.is_symlink():
+            if link_path.is_symlink():
+                # Check if it points to the right place
+                try:
+                    if link_path.resolve() == actual_path.resolve():
+                        print(f"  ✓  {plugin['name']} - link already exists")
+                        skipped += 1
+                        continue
+                    else:
+                        # Remove old symlink pointing elsewhere
+                        link_path.unlink()
+                except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)